blog/2022-09-30-mdx-blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: getting-started
+title: How to get started with sern!
+authors: [Sr Izan]
+tags: [guides]
+---
+
+## Hello everyone!
+
+I'm Sr Izan, your fellow sern-handler user and contributor.
+
+Today I'm going to show you how to get started with sern and all its cool features.
+
+### Step 1: Install the CLI.
+
+Just run:
+```
+npm i -g @sern/cli
+```
+then just use
+```
+sern init
+```
+The CLI is written in Typescript and open-sourced on [Github](https://github.com/sern-handler/cli). (thanks evo!)
+
+### Step 2: Have some way to store secrets.
+
+Normally you'd need a way to store secrets, and the best way to do that is by installing another package: `dotenv`
+
+just `npm i dotenv` in the project folder and add `require('dotenv').config()` to your import section. Then, when you login, `process.env.TOKEN` (or however you have it named on your `.env` file) should do the trick.
```

Comment:
Add an admonition saying require('dotenv') doesnt work for esm

Reply:
kay

Reply:
done